packages/cohort/src/tx_batch_executor.rs

```diff
@@ -93,15 +95,20 @@ impl BatchExecutor {
                 affected_rows += rows;
             } else {
                 // there was an error updating snapshot, we need to rollabck the whole batch
-                let snapshot_error = snapshot_update_result.unwrap_err();
```

Comment:
It seems lots of content around rollback has changed in this file. Why?

Reply:
Yes, the rollback for snapshot updates is removed, because this doesn't play well with parallel installs. 

Because you could have the snapshot installed while a higher version  was installed and if a lower version is trying to install, and with that code block, the entire transaction is rolled back for the lower version.